site/src/pages/OrganizationSettingsPage/IdpSyncPage/IdpSyncPageView.tsx

```diff
@@ -20,6 +20,10 @@ interface IdpSyncPageViewProps {
 	groupsMap: Map<string, string>;
 	roles: Role[] | undefined;
 	organization: Organization;
+	onGroupSyncFieldChange: (value: string) => void;
+	onRoleSyncFieldChange: (value: string) => void;
+	groupClaimFieldValues: string[] | undefined;
+	roleClaimFieldValues: string[] | undefined;
```

Comment:
```suggestion
	onGroupSyncFieldChange: (value: string) => void;
	groupClaimFieldValues: string[] | undefined;
	onRoleSyncFieldChange: (value: string) => void;
	roleClaimFieldValues: string[] | undefined;
```

nit: group by group, role by role. seems kinda nice, just a thought